components/common/ActionAttribute.tsx

```diff
@@ -17,30 +17,12 @@ import {
   ATTRIBUTE_WITH_NESTED_TYPE_FRAGMENT,
 } from '@/fragments/action-attribute.fragment';
 
-const ScaleIcon = styled(Icon)<{ $size?: 'sm' | 'md' }>`
-  font-size: ${({ $size }) => {
-    switch ($size) {
-      case 'sm':
-        return '.8em';
-      case 'md':
-        return '1.5em';
-      default:
-        return '1.5em';
-    }
-  }};
-
-  &.icon-on {
-    fill: ${({ theme }) =>
-      theme.actionAttributeColor ?? theme.brandDark} !important;
-  }
-
-  &.icon-off {
-    fill: ${(props) => props.theme.themeColors.light} !important;
-  }
-
-  &.icon-bad {
-    fill: ${(props) => props.theme.graphColors.red070} !important;
-  }
+const ScaleIcon = styled(Icon)<{ $active?: 'sm' | 'md' }>`
```

Comment:
Looks like the old type is sneaking in 🥷 

```suggestion
const ScaleIcon = styled(Icon)<{ $active?: boolean }>`
```

Reply:
Yes! Thanks